.circleci/config.yml

```diff
@@ -1462,7 +1449,7 @@ jobs:
             fi
           no_output_timeout: 5m
           environment:
-            ENABLE_CONFIRMATION_REDESIGN: "true"
```

Comment:
leave these drive-by changes out so we can `git blame` more usefully in the future.

Reply:
I think this was the formatter doing it automatically, I didn't even realize this was changed

Reply:
fixed